api.js

```diff
@@ -630,7 +635,7 @@ class Api {
         context.addDeferredHelp(this.initHelpBuffer())
       }
     } catch (err) {
-      context.unexpectedError(err)
+      context.unexpectedError(err, this.helpOpts.errorFormat ? this.helpOpts.errorFormat(err) : undefined)
```

Comment:
Instead of using the error formatter here, it seems like it would be better to configure the `context` with it and let the Context use it whenever `context.unexpectedError(err)` is called.

To hand the error formatter off to the context, let's modify `initContext` to something like this:

```js
initContext (includeTypes) {
  const context = this.get('_context', {
    utils: this.utils,
    pathLib: this.pathLib,
    fsLib: this.fsLib,
    errorFormatter: this._errorFormatter
  })
  return includeTypes ? this.applyTypes(context) : context
}
```